textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/field/ListField.java

```diff
@@ -7,15 +7,15 @@
 /**
  * Created by kishorenarendran on 25/04/16.
  */
-public class StringListField implements IField {
-    private final List<String> value;
+public class ListField<T> implements IField {
```

Comment:
Let's wait for the merge of https://github.com/TextDB/textdb/pull/77 (soon) before merging this PR.